### PR TITLE
Add skynet icon

### PR DESCRIPTION
**Pull request by arcyopterix**
**Issue:** Adding Skynet icon
**Alexa rank:** 101,719
  

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Adding the logo for Skynet found at https://siasky.net/ as requested in an Issue.
This is my first time contributing on this site, so if I've made any errors please let me know.
I'll be glad to correct them.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8226,6 +8226,11 @@
             "hex": "2FCEA0",
             "source": "https://www.skyliner.io/help"
         },
+        {
+            "title": "Skynet",
+            "hex": "00C65C",
```
Comment (kwypchlo) [inline _data/simple-icons.json:8231]: it should be `00C65E` according to https://support.siasky.net/key-concepts/skynet-brand-guidelines

> Skynet's primary color is #00C65E and uses two secondary colors.

**Comment by kwypchlo** [comment]
This pull request looked great, is there a reason you closed it @arcyopterix ?

**Comment by arcyopterix** [comment]
Hello, the closure was an accident actually. I deleted the branch after the pull request was accepted because I thought that was standard. It is my first pull request so sorry for the misunderstanding, please still make use of the contribution if possible. Thank you for the correction as well.

**Comment by kwypchlo** [comment]
Well the pull request haven't been merged - you can tell by looking at the original issue which is still open https://github.com/simple-icons/simple-icons/issues/6118

Would you like to restore the branch and change the color? I could do that by copying your contribution but it wouldn't feel fair since you did all the work already :)

**Comment by arcyopterix** [comment]
Sure, I will try it.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8258,6 +8258,11 @@
...
+            "source": "https://siasky.net/"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:8264]: This should probably also be updated to `https://support.siasky.net/key-concepts/skynet-brand-guidelines`, and we should definitely add that as a guidelines. 

However, it seems you used the SVG from the header of `https://siasky.net/` as source, is that correct? We will review the SVG after you answered this question.

```suggestion
            "source": "https://support.siasky.net/key-concepts/skynet-brand-guidelines",
            "guidelines": "https://support.siasky.net/key-concepts/skynet-brand-guidelines"
```

**Comment by ericcornelissen** [comment]
Thank you for jumping in and doing an initial review @kwypchlo! And thank you for doing your first ever Pull Request here @arcyopterix :smile: 

As @kwypchlo requested, you still need to update the hex value. I also added another request for a change (and a question). If you're stuck updating the Pull Request please do feel free to ask for help here :slightly_smiling_face:

File: _data/simple-icons.json (hunk 2 above)
Comment (arcyopterix) [reply _data/simple-icons.json:8264]: Hi, in this case I actually took the SVG from the footer of the homepage (https://siasky.net/). Does that change anything here?

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
I'm seeing some minor difference on the inside of [the top semicircle](https://user-images.githubusercontent.com/3742559/127713240-04818d35-355e-459b-bfc9-3da6aefef574.png) and [the bottom semicircle](https://user-images.githubusercontent.com/3742559/127713286-e2ca1d5c-7b37-4877-b882-024df59dc175.png) - regardless of whether I compare to an SVG found on [the branding page](https://support.siasky.net/key-concepts/skynet-brand-guidelines) (I used `Skynet Assets/Skynet Assets/Skynet logo.svg` in the `Skynet Assets.zip`) or the SVG in the footer of [the website](https://siasky.net/). It seems this is due to SVGO, so could you give it another shot with a higher precision and starting from the SVG from the branding page this time?

Also, don't forget to update the color and source :slightly_smiling_face:

File: _data/simple-icons.json (hunk 2 above)
Comment (ericcornelissen) [reply _data/simple-icons.json:8264]: Thanks for clarifying. Following my review of the SVG, can you update this as per my request? It's generally a better idea to use the actual brand guidelines as a source.

**Comment by service-paradis** [comment]
Closing this one in favor of #6367

Thank you @arcyopterix for your initial work on this!

**Pull request closed without merging**